docs/features/autoupdater.rst

```diff
@@ -11,14 +11,20 @@ during development), but it can be enabled by setting the variable GLUON_BRANCH
 to override the default branch set in the set in the site configuration.
 
 A manifest file for the updater can be generated with `make manifest`. A signing script (using
-ecdsautils) can by found in the `contrib` directory. When creating the manifest, ``GLUON_PRIORITY`` can
-be set on the command line, or it can be taken from the ``site.mk``.
+``ecdsautils``) can by found in the `contrib` directory. When creating the manifest, the variable 
+``PRIORITY`` in the manifest is set by the variable ``GLUON_PRIORITY`` which can be set on the
+command line or in ``site.mk``.
```

Comment:
This sentence is really ugly and repetitive. Also, PRIORITY is no variable (as the manifest is no script), but simply a value.

Maybe: "When creating the manifest, the \`\`PRIORITY\`\` value may be defined by setting \`\`GLUON_PRIORITY\`\` on the command line or in \`\`site.mk\`\`."